src/main/scala/io/getclump/package.scala

```diff
@@ -2,8 +2,6 @@ package io
 
 package object getclump {
 
-  private[getclump] implicit val executionContext = scala.concurrent.ExecutionContext.global
```

Comment:
All the tests pass in both versions: https://travis-ci.org/getclump/clump/builds/70654028